src/main/java/seedu/address/model/wedding/Wedding.java

```diff
@@ -20,11 +20,12 @@ public class Wedding {
      * Constructs a {@code Wedding}.
      */
     public Wedding(Husband husband, Wife wife, Date date, Venue venue) {
-        requireAllNonNull(husband, wife);
+        requireAllNonNull(husband, wife, date, venue);
```

Comment:
i thought that date and venue can be undecided initially so they can be instantiated to null